claims_hosp/HospClaims/automate/hosp_claims_regen_script.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/sh
```

Comment:
This script is similar to `hosp_claims_master_script.sh`, it just has very small modifications to correct timestamps and such when regenerating old issues. Not sure if other indicators have a different way to be regenerated? If so, this can be deleted.